src/lib389/lib389/__init__.py

```diff
@@ -1071,7 +1104,13 @@ def start(self, timeout=120, post_open=True):
         if self.with_systemd():
             self.log.debug("systemd status -> True")
             # Do systemd things here ...
-            subprocess.check_output(["systemctl", "start", "dirsrv@%s" % self.serverid], stderr=subprocess.STDOUT)
+            try:
+                subprocess.check_output(["systemctl", "start", "dirsrv@%s" % self.serverid], stderr=subprocess.STDOUT)
+            except subprocess.CalledProcessError as e:
+                self.dump_errorlog()
+                logger.info('Failed to start dirsrv@%s: "%s"' % (self.serverid, e.output.decode()))
```

Comment:
FYI: Using logger.info to logs data here because self.log.info and logger.errors fails in `dsctl instance start`
          [1] self.log is not yet initialized
          [2] error log level is not enabled.  (only INFO (and DEBUG if -v option is used) are set)  and nothing is logged.